docs/source/reference/leaderboard_faq.rst

```diff
@@ -118,7 +118,11 @@ The formula for the total score is subject to change as additional performance m
 The leaderboard in the explorer scales this metric and displays it as a value in the range 0-10.
 
 As an orchestrator operator, how can I improve my metrics on the leaderboard?
-*******************************************************************************
+
```

Comment:
Note that the "****..." is required to create the proper headings in an .rst file.